kroxylicious-kms/src/main/java/io/kroxylicious/kms/service/KekId.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.service;
+
+/**
+ * A semantic wrapper around KMS specific key id type
+ */
+public interface KekId {
```

Comment:
I've gone with an interface for now as I'm expecting to add further behaviour to the KekId however I am starting to wonder if a record would be more appropraite.

Reply:
A quick look at using a record leads the the viral re-introduction of generics so I'm parking that thought for now.